llm_studio/python_configs/text_causal_language_modeling_config.py

```diff
@@ -158,16 +174,24 @@ def __post_init__(self):
         self._possible_values[
             "differential_learning_rate_layers"
         ] = possible_values.String(
-            values=["encoder", "decoder"],
+            values=["backbone", "value_head"],
             allow_custom=False,
             placeholder="Select optional layers...",
         )
         self._possible_values["differential_learning_rate"] = self._possible_values[
             "learning_rate"
         ]
+        self._possible_values["reward_model"] = possible_values.String(
+            values=(
+                "OpenAssistant/reward-model-deberta-v3-large-v2",
+                "OpenAssistant/oasst-rm-2.1-pythia-1.4b-epoch-2.5",
+                "OpenAssistant/oasst-rm-2-pythia-6.9b-epoch-1",
+            ),
+            allow_custom=True,
```

Comment:
`allow_custom=False` as `get_score` method currently only supports the models listed above.